nature.md

```diff
@@ -1352,7 +1354,7 @@ Notes: This product has not been evaluated for WCAG 2.x Level AAA conformance.
 <tr>
 <td>302.3 Without Perception of Color</td>
 <td>Supports</td>
-<td>Color perception is not needed to use this product.</td>
+<td>Color perception is not generally needed to use this product.</td>
```

Comment:
Do we want to say more about our aims and/or internal practices on this point?

Reply:
Not here because it's fully described in the WCAG section above, as is the case with all the 508 criteria - the WCAG table is just more granular so it's easier to be explicit about the issues. 

I could copy it over, but it'd just be repetition 😄